src/App.js

```diff
@@ -25,6 +25,7 @@ const App = () => {
   const filteredPosts = filterPosts(items, searchQuery);
   const [show, setShow] = useState(false)
   const [deleteUser, setDeleteUser] = useState(0)
+  const [robotName, setRobotName] = useState("")
```

Comment:
This variable name should be more meaningful, something like `robotToDelete` or something, just `robotName` might be a bit too vague to people reading the code.
Although, it is actually possible to pass the robot name to the modal without using this state. Have a think about what else we can recycle :)